compiler/rustc_codegen_gcc/build_system/build_sysroot/Cargo.toml

```diff
@@ -16,6 +15,7 @@ proc_macro = { path = "./sysroot_src/library/proc_macro" }
 rustc-std-workspace-core = { path = "./sysroot_src/library/rustc-std-workspace-core" }
 rustc-std-workspace-alloc = { path = "./sysroot_src/library/rustc-std-workspace-alloc" }
 rustc-std-workspace-std = { path = "./sysroot_src/library/rustc-std-workspace-std" }
+compiler_builtins = { path = "./sysroot_src/library/compiler-builtins/compiler-builtins" }
```

Comment:
@antoyo It should be possible to remove `build_sysroot/Cargo.toml` and use `sysroot_src/library/sysroot/Cargo.toml` instead. That is what cg_clif does nowadays too.